src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -317,6 +318,50 @@ SyncCitusTableMetadata(Oid relationId)
 		ObjectAddressSet(relationAddress, RelationRelationId, relationId);
 		MarkObjectDistributed(&relationAddress);
 	}
+
```

Comment:
Could you please enhance the comment of the function `SyncCitusTableMetadata` to include why we are creating views as a part of it.